Makefile

```
@@ -1,2 +1,19 @@
 lint:
 	golangci-lint run
+
+clean:
+    # '-f' to ignore when 'candebot' is not found
+	rm -f candebot 
+
+get-dependencies:
+	echo "Getting dependencies..."
+	go get -v -t -d ./...
+
+fast-build:
+	go build -v .
+
+build: clean get-dependencies fast-build
+	echo "'candebot' built and ready to serve and protect."
```

The get-deps step is not needed, as go automatically deals with that when building or installing. See https://github.com/bcneng/candebot/pull/61.

So I think we can simplify to:

```suggestion
build: clean
	go build -v .
	@echo candebot built and ready to serve and protect.
```

p.s notice I prepended the echo with `@` so it's not printed twice, see https://www.gnu.org/software/make/manual/html_node/Echoing.html

also there is no var interpolation so no double quoting needed I guess